feature_engine/encoding/one_hot.py

```diff
@@ -284,3 +286,22 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
     def inverse_transform(self, X: pd.DataFrame):
         """inverse_transform is not implemented for this transformer."""
         return self
+
+    def get_feature_names(self) -> List[str]:
+        """
+        Return feature names for output features.
+
+        -------
+        feature_names : list of str of shape (n_output_features,) of feature names
+        """
+        check_is_fitted(self)
+
+        feature_names = []
+        for feature in self.input_features_:
```

Comment:
Oh, I see... you want to have the parameter input_features_ in fit, and then add those features that are added or removed in the function get_feature_names(). Yes, I like this better.